Formula/miniserve.rb

```diff
@@ -15,8 +14,8 @@ class Miniserve < Formula
 
   # Miniserve requires a known-good Rust nightly release to use.
   resource "rust-nightly" do
-    url "https://static.rust-lang.org/dist/2019-08-24/rust-nightly-x86_64-apple-darwin.tar.xz"
-    sha256 "104ddea51b758f4962960097e9e0f3cabf2c671ec3148bc745344431bb93605d"
+    url "https://static.rust-lang.org/dist/2020-03-14/rust-nightly-x86_64-apple-darwin.tar.xz"
+    sha256 "6cfe5b598502d4bc6afb1bec3b1e87306d3b4057ce1ffcd8a306817c2ff5fc87"
   end
```

Comment:
I don't think this is needed anymore (https://github.com/svenstaro/miniserve/pull/241) and could just depend on Rust.

Reply:
let me remove to see if it can build again.

Reply:
Yeah, looks like not quite working
```
   Compiling maud_macros v0.21.0
error[E0554]: `#![feature]` may not be used on the stable release channel
 --> /Users/ruichen/Library/Caches/Homebrew/cargo_cache/registry/src/github.com-1ecc6299db9ec823/maud_macros-0.21.0/src/lib.rs:1:1
  |
1 | #![feature(proc_macro_def_site)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error[E0554]: `#![feature]` may not be used on the stable release channel
 --> /Users/ruichen/Library/Caches/Homebrew/cargo_cache/registry/src/github.com-1ecc6299db9ec823/maud_macros-0.21.0/src/lib.rs:2:1
  |
2 | #![feature(proc_macro_diagnostic)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error[E0554]: `#![feature]` may not be used on the stable release channel
 --> /Users/ruichen/Library/Caches/Homebrew/cargo_cache/registry/src/github.com-1ecc6299db9ec823/maud_macros-0.21.0/src/lib.rs:3:1
  |
3 | #![feature(proc_macro_hygiene)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error[E0554]: `#![feature]` may not be used on the stable release channel
 --> /Users/ruichen/Library/Caches/Homebrew/cargo_cache/registry/src/github.com-1ecc6299db9ec823/maud_macros-0.21.0/src/lib.rs:4:1
  |
4 | #![feature(proc_macro_quote)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error[E0554]: `#![feature]` may not be used on the stable release channel
 --> /Users/ruichen/Library/Caches/Homebrew/cargo_cache/registry/src/github.com-1ecc6299db9ec823/maud_macros-0.21.0/src/lib.rs:5:1
  |
5 | #![feature(proc_macro_span)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to 5 previous errors
```

Reply:
Their stable/beta builds always failed.
https://travis-ci.org/github/svenstaro/miniserve/builds/662414072

Reply:
:+1: Thanks for checking

Reply:
 np